build-system/tasks/visual-diff/index.js

```diff
@@ -662,13 +663,13 @@ async function createEmptyBuild() {
   const browser = await launchBrowser();
   const page = await newPage(browser);
 
-  await page
-    .goto(`http://${HOST}:${PORT}/examples/visual-tests/blank-page/blank.html`)
-    .then(
-      () => {},
-      () => {}
+  try {
+    await page.goto(
+      `http://${HOST}:${PORT}/examples/visual-tests/blank-page/blank.html`
     );
-  await percySnapshot(page, 'Blank page', SNAPSHOT_SINGLE_BUILD_OPTIONS);
+  } finally {
```

Comment:
```javascript
try {
  await page.goto(...
} catch {  // <-- if you don't include this catch clause, exceptions will bubble up!
  // Ignore failures
}

// <-- no need for a finally now
await percySnapshot(page, 'Blank page', SNAPSHOT_SINGLE_BUILD_OPTIONS);
```

Reply:
Yep! If you run the following code, "1" would be logged in the console, and then it will throw Error(123) up the stack:
```js
try {
  throw new Error('123');
} finally {
  console.log(1);
}
```

This is similar to doing:
```js
new Promise((resolve, reject) => {
  reject('123');
}).finally(() => {
  console.log(1);
});
```

On the other hand, if you do the following it will log "1" to the console and then silently ignore the error:
```js
try {
  throw new Error('123');
} catch {
  // Do nothing.
} finally {
  console.log(1);
}
```

This is equivalent to:
```js
new Promise((resolve, reject) => {
  reject('123');
}).catch(() => {
  // Do nothing.
}).finally(() => {
  console.log(1);
});
```

Reply:
:man_facepalming: Sorry, this was me forgetting the basics of try/catch/finally at the end of a long week. Will fix

Reply:
Done